src/js/services/forms.js

```diff
@@ -235,18 +237,22 @@ export default App.extend({
     Promise.all([
       Radio.request('entities', 'fetch:forms:data', get(this.action, 'id'), this.patient.id, this.form.id),
       this.fetchLatestFormResponse(),
-    ]).then(([data, response]) => {
-      this.send(message, { value: {
-        isReadOnly: this.isReadOnly(),
-        formData: data.attributes,
-        responseData: response.getFormData(),
-        formSubmission: response.getResponse(),
-        options: this.form.get('options'),
-      } }, requestId);
-    }).catch(({ responseData }) => {
-      /* istanbul ignore next: Don't test BE errors */
-      this.send(message, { error: responseData }, requestId);
-    });
+    ])
```

Comment:
the changes in this file are just style / moving comments.. coveralls doesn't count the function hits.. only lines/branches.. moving these inside the catch make it such that the local coverage reports are all 100%